sql/synth_recipes.sql

```diff
@@ -2816,7 +2816,7 @@ INSERT INTO `synth_recipes` VALUES (41513,0,2016,0,0,0,0,35,0,0,0,4103,4245,635,
 INSERT INTO `synth_recipes` VALUES (41514,0,2018,0,0,0,0,35,0,0,0,4103,4245,635,635,635,859,859,859,2129,4509,851,851,851,851,3,3,3,3,'Ram Leather');
 INSERT INTO `synth_recipes` VALUES (41515,0,0,0,0,0,0,35,0,0,0,4103,4245,635,859,4509,0,0,0,0,0,851,851,851,851,1,1,1,1,'Ram Leather');
 INSERT INTO `synth_recipes` VALUES (41516,0,0,0,0,0,0,35,0,0,0,4099,4241,850,12882,0,0,0,0,0,0,14265,14266,14266,14266,1,1,1,1,'San d\'Orian Trousers');
-INSERT INTO `synth_recipes` VALUES (41517,1,0,0,0,0,0,35,0,0,0,4098,4240,1624,0,0,0,0,0,0,0,851,823,823,823,3,2,2,2,'Bugbear Mask (desynth)');
+INSERT INTO `synth_recipes` VALUES (41517,1,0,0,0,0,0,35,0,0,0,4098,4240,1624,0,0,0,0,0,0,0,851,823,823,823,3,3,3,2,'Bugbear Mask (desynth)');
```

Comment:
This is correct regarding the quantity changes, but is incorrect regarding the items rewarded.

Item ID 851 (Ram Leather) should be rewarded when NQ, HQ1 and HQ2
Item ID 823 (Gold Thread) should be rewarded when HQ3

So the change should be this...

```sql
INSERT INTO `synth_recipes` VALUES (41517,1,0,0,0,0,0,35,0,0,0,4098,4240,1624,0,0,0,0,0,0,0,851,851,851,823,3,3,3,2,'Bugbear Mask (desynth)');
```

Reply:
Thanks for catching that! Fixed.